src/globus_cli/commands/flows/start.py

```diff
@@ -250,16 +253,46 @@ def start_command(
 
     flow_client = login_manager.get_specific_flow_client(flow_id)
     auth_client = login_manager.get_auth_client()
-
-    response = flow_client.run_flow(
-        body=input_document_json,
-        label=label,
-        tags=list(tags),
-        run_managers=list(managers),
-        run_monitors=list(monitors),
-        activity_notification_policy=notify_policy,
-    )
+    flow_scope = flow_client.scopes.user.scope_string
+
+    with scope_injected_into_raised_gares(flow_scope):
+        response = flow_client.run_flow(
+            body=input_document_json,
+            label=label,
+            tags=list(tags),
+            run_managers=list(managers),
+            run_monitors=list(monitors),
+            activity_notification_policy=notify_policy,
+        )
 
     fields = flow_run_format_fields(auth_client, response.data)
 
     display(response, fields=fields, text_mode=display.RECORD)
+
+
+@contextlib.contextmanager
+def scope_injected_into_raised_gares(scope: str) -> t.Iterator[None]:
+    """
+    A context manager which catches GARE-convertible GlobusAPIErrors and reraises
+    as an exact copy of the original GARE but with the supplied scope injected.
+
+    Due to SDK Error mutability limitations, these GAREs are raised as
+    CLIAuthRequirementsErrors. All other categories of errors raise as normal.
+
+    :param scope: The scope to inject into any caught GAREs.
+    :raises CLIAuthRequirementsError: with modified GARE if a GARE-compatible
+        GlobusAPIError is caught.
+    """
+
+    try:
+        yield
+    except globus_sdk.GlobusAPIError as e:
+        gare = to_gare(e)
+        if gare:
+            scopes = gare.authorization_parameters.required_scopes or []
+            if scope not in scopes:
+                scopes.append(scope)
+
+            gare.authorization_parameters.required_scopes = scopes
+            raise CLIAuthRequirementsError("", gare=gare)
```

Comment:
Because this is being caught and transformed, I think this should be a `raise ... from None`. Thoughts?

```suggestion
            raise CLIAuthRequirementsError("", gare=gare) from None
```